cmd/immuadmin/command/database.go

```diff
@@ -111,19 +118,26 @@ func (cl *commandline) database(cmd *cobra.Command) {
 				c.PrintfColorW(cmd.OutOrStdout(), c.Yellow, "Replication is a work-in-progress feature. Not ready for production use\n")
 			}
 
+			excludeCommitTime, err := cmd.Flags().GetBool("exclude-commit-time")
```

Comment:
I'm wondering if there will ever be a need to update this flag - either you want to have reproducible hashes and should start with no timestamps from the beginning or you start with timestamps and don't disable it later (because you won't have reproducibility anyway due to initial commits with timestamps). It could simplify the rest of the code I believe?

Reply:
The reason to even naming the flag `exclude-commit-time` instead of `reproducible` it's because in order to fully ensure reproducibility there may be other things in consideration. Also making it explicit on the API to dynamically change it may help in understanding that reproducibility may be obtained if server commit time was always excluded. But in order to prevent changes, there must be logic to validate it never happen. Which may be out of the initial scope